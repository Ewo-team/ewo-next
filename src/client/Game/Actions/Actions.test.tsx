import React from 'react';
import renderer from 'react-test-renderer';
import { Actions } from '.';

describe('UI Actions', () => {

  describe('Actions', () => {

    it('should render', () => {

      const component = renderer.create(
        (
          <Actions />
        ),
      );

      const tree = component.toJSON();
      expect(tree).toMatchSnapshot('render');
    });

  });

});
