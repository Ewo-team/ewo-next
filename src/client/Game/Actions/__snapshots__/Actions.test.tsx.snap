// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UI Actions Actions should render: render 1`] = `
<div
  className="Actions Game__Container"
>
  <div
    className="Title"
  >
    Actions
  </div>
</div>
`;
